Util/UserManipulator.php

```diff
@@ -45,10 +45,6 @@ class UserManipulator
 
     /**
      * UserManipulator constructor.
```

Comment:
this could be removed too, as this short description does not provide any value.